starter-code/index.html

```diff
@@ -5,8 +5,92 @@
+  <nav>
+    <div class="top-left">
+      <a class = "heart" href="#">♥︎</a>
+      <span class="acronym">Neophobe Plebeian Mumpsimus</span>
+    </div>
+    <ul class="top-links">
+      <li><a href="#">npm Enterprise</a></li>
+      <li><a href="#">features</a></li>
+      <li><a href="#">pricing</a></li>
+      <li><a href="#">documentation</a></li>
+      <li><a href="#">support</a></li>
+    </ul>
+  </nav>
+  <header>
+    <div class="header-top">
+      <img class="bear" src="images/bear-logo.png" alt="bear">
+      <form action="#" class="bar">
+        <input type="text" placeholder="Search packages" name="search" class="search-bar">
+        <button type="submit" class="search-button"><i class="fa fa-search"></i></button>
+        <a href="#" class="header-links">log in</a>
+        or
+        <a href="#" class="header-links">sign up</a>
+      </form>
+      <img class="logo" src="images/npm-logo.png" alt="NPM logo">
+    </div>
+    <div class="build">
+      <h1>Build amazing things</h1>
+      <p>npm is the package manager for JavaScript and the world’s largest<br> software registry. Discover packages of reusable code — and<br> assemble them in powerful new ways.</p>
+      <form action="#">
+        <button type="submit" class="orange-button">Sign up</button>
+      </form>
+    </div>
+  </header>
+  <section id="section1">
+    <img src="images/collaboration.svg" alt="Collaboration" class="collaboration">
+    <div class="section1">
+      <h1>npm Orgs is powerful <br>collaboration — for free</h1>
+      <ul class="collaboration-list">
+        <li>Encourage code discovery and re-use within teams</li>
+        <li>Publish and control access to your own namespace</li>
+        <li>Manage public and private code with the same workflow</li>
+      </ul>
+      <button type="submit" class="red-button">Sign up for Orgs</button><br>
+      <br>or,
+      <a href="#" class="learn-link">Learn more about Orgs</a>
+    </div>
+  </section>
+  <section id="section2">
+    <div class="background">
+      <div class="section2">
+        <h1>What is npm?</h1>
+        <p>Use npm to install, share, and distribute code; manage dependencies in your projects; and share & receive feedback with others.</p>
+        <button type="submit" class=grey-button>Install npm</button>
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class=spacer>
+      <p></p>
+    </div>
+  </section>
+  <section id="section3">
+    <div class="section3text">
+      <h1>What can you make with 800,000 building blocks?</h1>
+      <p>The npm registry hosts the world’s largest collection of free, reusable code.</p>
+    </div>
+    <div class="div">
+      <img src="images/binoculars.svg" alt="Binoculars">
+      <h3>Find</h3>
+      <p>Libraries like <a href="#">jQuery</a>, <a href="#">Bootstrap</a>, <a href="#">React</a>, and <a href="#">Angular</a>, and components from frameworks such as <a href="#">Ember</a>.</p>
+    </div>
+    <div div class="div">
```

Comment:
What is this extra div doing here? 'div' as a class name is a bit confusing.

Reply:
Must be a copy-paste fail from the first div of the section 
```suggestion
    <div class="div">
```